### ISSUE TITLE
failing for "+..34"

### ISSUE THREAD

**Issue by harendra247** [cruxrebels/InterviewBit#17]

**Comment by cruxrebels** [comment]
Hi @harendra247, could you please provide for which code file or line is this issue opened? 

Also, if it's failing on the InterviewBit online judge please attach a screenshot of the failure.

**Comment by harendra247** [comment]
My bad! It is ValidNumber.cpp  try running it for "+..34" input

**Comment by cruxrebels** [comment]
You're right. The Strings problem `ValidNumber.cpp` can be enhanced to handle `+..34` case since expected result is `0` and the solution returns `1` currently. 

Do you want to raise a fix and contribute for this solution?
